src/CollAction/Views/Projects/Details.cshtml

```diff
@@ -89,6 +89,5 @@
     </dl>
 </div>
 <div>
-    <a asp-action="Edit" asp-route-id="@Model.Project.Id">@Localizer["Edit"]</a> |
```

Comment:
I'm assuming this was added because it caused issues with users not logged in? Which is fine btw.